### DISCUSSION TITLE
prompt says can't modify .zshrc file when configuring powerlevel10k

### DISCUSSION THREAD

**Discussion by 1NC4NDESCENCE** [ohmyzsh/ohmyzsh#10123] [Help] [answered]
When about to finish configuring powerlevel10k on my terminal, it said it can't modify the .zshrc file and I tried to change the permission setting with chmod command and it didn't work. Also, my .zshrc file seems pretty different from others'. Below are my result when running ls -l .zshrc and the content in the .zshrc file.
```
-rw-r--r--  1 root  staff  509 Aug 18 22:02 .zshrc
```
```
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/opt/miniconda3/bin/conda' 'shell.zsh' 'hook' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/opt/miniconda3/etc/profile.d/conda.sh" ]; then
        . "/opt/miniconda3/etc/profile.d/conda.sh"
    else
        export PATH="/opt/miniconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
source /usr/local/opt/powerlevel10k/powerlevel10k.zsh-theme
```
The last line was added by me to manually change the theme of zsh.
As you can see, it is pretty short and has something to do with conda, I uninstalled conda and it didn;t change anything.

**Comment by rwmitchell** [comment, marked as answer]
It failed because .zshrc is owned by root. It should be owned by you (user). This is usually from using sudo unnecessarily.
